WhaleSpotting/react-app/src/styles/Navbar.scss

```
@@ -0,0 +1,68 @@
+@import './constants.scss';
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.header {
+    position: sticky;
+    top: 0;
+    width: 100%;
+    background-color: $primaryColour;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: $headerHeight;
+    z-index: 10;
+}
+
+.nav-bar {
+    display: flex;
+    width: 100%;
+    max-width: 1024px;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.navbar-link {
+    margin-left: 5%;
+    margin-right: 5%;
+    padding: 3%;
+    white-space: nowrap;
+    color: white;
+    text-decoration: none;  
+    font-size: calc(10px + 1vmin);  
+}
+
+.navbar-link:hover {
```

```suggestion
    &:hover {
```
You can merge these slightly